stellargraph/core/element_data.py

```diff
@@ -107,6 +107,8 @@ class ElementData:
         shared (dict of type name to pandas DataFrame): information for the elements of each type
     """
 
+    _SHARED_REQUIRED_COLUMNS = []
```

Comment:
On #776, @timpitman said:

> I know this is called "shared" but possibly an extra comment to say that it's sometimes overridden in derived classes?

The "shared" here is referring to the `shared` argument to `__init__`. Or is this not what you're getting at?

Reply:
I was referring to `_SHARED_REQUIRED_COLUMNS`, suggesting a comment that it may be overridden in derived classes

Reply:
Right, that's what the comment is attached to. I'm more asking about your quoting of "shared" in the original comment: that word isn't referring to being shared among subclasses, but the argument to `__init__` that's called `shared`.

I'll add a comment, but I personally generally don't explain things like this when no-one should need to think about it outside a single file (that is, no other part of the library needs to know about this variable and its functionality, and certainly no external users: they should barely even know that `ElementData` and its subclasses exist).